code/defines/obj.dm

```diff
@@ -99,6 +99,17 @@
 	"}
 	var/even = 0
 	// sort mobs
+	if(OOC)
+		for(var/mob/living/silicon/ai/dooropener in mob_list)
+			bot[dooropener.name] = "AI"
+			isactive[dooropener.name] = (dooropener.client?.inactivity > 10 * 60 * 10) ? "Active" : "Inactive"
+		for(var/mob/living/silicon/robot/tincan in mob_list)
+			if(isMoMMI(tincan))
+				continue
+			if(tincan.syndicate)
+				continue
+			bot[tincan.name] = "Cyborg"
```

Comment:
Isn't 'Android' used when borgs are created with Posibrains? Might want to catch that one too.